now.json

```
@@ -2,14 +2,18 @@
   "name": "paste",
   "scope": "twilio-dsys",
   "version": 2,
-  "builds": [{
-    "src": "now-build.sh",
-    "use": "@now/static-build",
-    "config": {
-      "distDir": "packages/paste-website/public"
+  "trailingSlash": false,
```

```suggestion
  "trailingSlash": true,
```

I think we want the trailing slash. It looks like setting it `true` will forward any non-slash urls to slash urls.

---

👍